pymilvus/client/grpc_handler.py

```diff
@@ -1546,13 +1546,19 @@ def load_balance(
         check_status(status)
 
     @retry_on_rpc_failure()
-    def compact(self, collection_name: str, timeout: Optional[float] = None, **kwargs) -> int:
+    def compact(
+        self,
+        collection_name: str,
+        timeout: Optional[float] = None,
+        is_clustering: Optional[bool] = False,
```

Comment:
Please leave timeout as the last param. And check is_clustering bool type in Prepare